immuni_exposure_ingestion/core/config.py

```diff
@@ -59,7 +59,12 @@
     "DUMMY_DATA_TOKEN_ERROR_CHANCE_PERCENT", cast=int, default=1
 )
 
-MAX_KEYS_PER_UPLOAD: int = config("MAX_KEYS_PER_UPLOAD", cast=int, default=14)
+# [14 days before TEKs + (possibly) 1 current day TEK with rolling_period up until "now" = 15 TEKs]
+# is the lower bound.
+# Some slack is given in case the underlying Apple and Google exposure notification frameworks start
+# allowing rolling TEKs more often (rolling_period < 144, more than a TEK per day).
+# We here assume two TEKs per day.
```

Comment:
the client should generate more than 1 tek per day only in case it is submitting its teks for being tested positive. Which is the use case when we expect the client to generate 2 teks per day in the last 14 days?

Reply:
I was thinking more of a case when the client tries to upload multiple times (and the first tries are unsuccessful).

Reply:
The above is a valid usecase, although quite uncommon. At the same time, the "spare" padding space seems sufficient to hold these additional 15 TEKs.

Reply:
Considering the case when the client tries to upload multiple times (and the first tries are unsuccessful) having very little chances of happening, and having logs to spot it, we remain strict, for now.